README-local.md

```diff
@@ -0,0 +1,298 @@
+
+```
+sudo apt update
+sudo apt install nginx
+sudo ufw allow 'Nginx HTTP'
+```
+
+To test
+
+```
+systemctl status nginx
+```
+
+Example Expected output
+
+```
+● nginx.service - A high performance web server and a reverse proxy server
+   Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
+   Active: active (running) since Tue 2020-09-15 13:41:04 EDT; 1min 59s ago
+     Docs: man:nginx(8)
+ Main PID: 23138 (nginx)
+    Tasks: 7 (limit: 4662)
+   CGroup: /system.slice/nginx.service
+           ├─23138 nginx: master process /usr/sbin/nginx -g daemon on; master_process on;
+           ├─23140 nginx: worker process
+           ├─23142 nginx: worker process
+           ├─23144 nginx: worker process
+           ├─23145 nginx: worker process
+           ├─23147 nginx: worker process
+           └─23149 nginx: worker process
+
+... Starting A high performance web server and a reverse proxy server...
+... Started A high performance web server and a reverse proxy server.
+```
+
+## Install PostGres
+
+*For more detailed instructions, go [here](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-18-04)*
+
+```
+sudo apt install postgresql postgresql-contrib
+```
+
+Configure postgres
+
+```
+sudo -u postgres psql ## login to postgres
+create database talentcloud; ### create database
+create user talentcloud with encrypted password 'talentcloud'; ### create user
+grant all privileges on database talentcloud to talentcloud; ### adjust access
+ALTER ROLE talentcloud SUPERUSER;
+```
+
+## Install PHP
+
+*For more detailed instructions, [go here](https://www.digitalocean.com/community/tutorials/how-to-install-linux-nginx-mysql-php-lemp-stack-ubuntu-18-04)*
+
+```
+sudo add-apt-repository universe
+sudo apt install php-fpm php7.2-pgsql
+```
+
+
+## Configure nginx
+
+*For more detailed instructions, [go here](https://www.digitalocean.com/community/tutorials/how-to-install-linux-nginx-mysql-php-lemp-stack-ubuntu-18-04)*
+
+```
+sudo vi /etc/nginx/sites-available/talent.test
+sudo ln -s /etc/nginx/sites-available/talent.test /etc/nginx/sites-enabled/
+sudo unlink /etc/nginx/sites-enabled/default
+sudo nginx -t ## to est new config
+sudo mkdir /var/www ## create the folder we will copy our files to, and tell nginx to point to for root
+```
+
+Copy in the below config into /etc/nginx/sites-available/talent.test (borrowed from [here](https://laravel.com/docs/7.x/deployment)). The only difference between the config from the docs and the one below is changing 'php7.4-fpm.sock' => 'php7.2-fpm.sock'
+
+```
+server {
+
+    listen 80;
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+    server_name talent.test;
+
+    ssl_certificate /etc/ssl/certs/localhost.crt;
+    ssl_certificate_key /etc/ssl/private/localhost.key;
+
+    ssl_protocols TLSv1.2 TLSv1.1 TLSv1;
+
+    root /var/www/public;
```

Comment:
In case you don't want to bother having to copy things locally, you can set your root to the project directory and just serve out of there directly.

For example, my local nginx config on OSX has
```
    root /Users/{username}/Projects/TalentCloud/public;
```

Reply:
I've just been developing directly out of the /www/var directory.